scripts/src/create_data/offers.ts

```diff
@@ -130,18 +136,24 @@ export async function createEarn(
 		});
 	}
 
-	offer.amount = amount;
-	offer.meta = {
-		title, image, description,
-		order_meta: {
-			title: orderTitle,
-			description: orderDescription,
-		}
-	};
-	!options.dryRun && await offer.save();
+	if (options.onlyUpdateMetaImage) {
+		offer.meta.image = image;
+	} else {
+		offer.amount = amount;
+		offer.meta = {
+			title, image, description,
```

Comment:
i think that if the object spans multiple line then better give each property it's own like